test/acceptance/docker-compose.override.yaml

```diff
@@ -0,0 +1,19 @@
+version: '3.8'
+services:
+  ffc-pay-batch-processor-acceptance-tests:
+    build:
+      target: development
+    image: ffc-pay-batch-processor-acceptance-tests
+    container_name: ffc-pay-batch-processor-acceptance-tests
+    ports:
+      - "9333:9229"
+    volumes:
+      - ./features:/home/node/features
+      - ./step_definitions:/home/node/step_definitions
+      - ./package.json:/home/node/package.json
+      - ./support:/home/node/support
+      - ./reports:/home/node/reports
+      - ./cucumber.js:/home/node/cucumber.js
```

Comment:
Shouldn't need to map this granualar.  Also suggest binding reports to existing `test-output` folder.

However, see other comments about docker compose setup.